src/lib/viewers/BaseViewer.js

```diff
@@ -1037,6 +1040,27 @@ class BaseViewer extends EventEmitter {
         this.annotator.scrollToAnnotation(data);
     }
 
+    /**
+     * Handles retrieving the active annotation id from a deep-linked annotation and scrolls to the annotation.
+     * @param {Object} event - annotations array from emitted from box-annotations.
+     * @param {Array} event.annotations - annotations array from emitted from box-annotations.
+     * @return {void}
+     */
+    handleAnnotationsInitialized = ({ annotations = [] }) => {
```

Comment:
Is `event` always defined? Should we guard against it being null or undefined?

Reply:
For now it should always be defined from annotations.